lib/diagnostic.rb

```diff
@@ -49,31 +50,31 @@ def method_name
 
 # define your method here
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = odd?
 
```

Comment:
Okay.. I didn't read the question fully. I didn't know we were suppose to write a method out.